### PR TITLE
Noticed some updates needed re web console

### PR DESCRIPTION
**Pull request by sosiouxme**

### CODE REVIEW

File: architecture/infrastructure_components/web_console.adoc [hunk 1]
```diff
@@ -15,17 +15,16 @@ Developers can use the web console to visualize, browse, and manage the contents
 of link:../../dev_guide/projects.html[projects].
 
 The web console is started as part of the
-link:kubernetes_infrastructure.html#master[master] and by default listens one
-port higher than the API server. All static assets required to run the web
```
Comment (sosiouxme) [inline architecture/infrastructure_components/web_console.adoc]: Now listens at the same port (at `/console` if that matters).

File: architecture/infrastructure_components/web_console.adoc [hunk 2]
```diff
@@ -15,17 +15,16 @@ Developers can use the web console to visualize, browse, and manage the contents
...
+link:kubernetes_infrastructure.html#master[master]. All static assets required to run the web
 console are served from the `openshift` binary.
 
 When you access the web console from a browser, it first loads all required
-static assets. It then makes requests to the OpenShift and Kubernetes APIs using
-the values defined from the `openshift start` options `--public-kubernetes` and
```
Comment (sosiouxme) [inline architecture/infrastructure_components/web_console.adoc]: Openshift serves as proxy for kube requests; I don't think there's any use case for `--public-kubernetes` at this point (if there ever was).

File: architecture/infrastructure_components/web_console.adoc [hunk 3]
```diff
@@ -38,10 +37,12 @@ link:http://caniuse.com/#feat=websockets[WebSockets].
 .Web Console Request Architecture
 image::web_console_request_arch.png["Web Console Request Architecture"]
 
-The default host name for the web console is whitelisted to communicate with the
-API server. To access the web console using a different host name, you must
+The configured host names and IP addresses for the web console are whitelisted to access the
+API server safely even when the browser would consider the requests to be link:http://www.w3.org/TR/cors/[cross-origin]. To access the API server from a web application using a different host name, you must
```
Comment (sosiouxme) [inline architecture/infrastructure_components/web_console.adoc]: The original issue was that the console being at a different port was considered cross-origin (requests for JSON at API considered a different origin from the web application at a different port). With the web console now at the same origin it doesn't matter as much. But could still be applicable if accessing the web console at a different location from the publicMasterURL for some reason, or for 3rd-party web apps that use our APIs.

File: architecture/infrastructure_components/web_console.adoc (hunk 1 above)
Comment (adellape) [inline architecture/infrastructure_components/web_console.adoc]: Should we mention `/console` here then? Currently the only reference as to what to hit from a browser is in:

http://docs.openshift.com/enterprise/latest/dev_guide/authentication.html#web-console-authentication

File: architecture/infrastructure_components/web_console.adoc [hunk 4]
```diff
@@ -38,10 +37,12 @@ link:http://caniuse.com/#feat=websockets[WebSockets].
...
 whitelist that host name by specifying the `--cors-allowed-origins` option
-during `openshift start`.
+on `openshift start` or from the related
+link:../../admin_guide/master_node_configuration.html#master-configuration-files[master
+configuration file parameter `corsAllowedOrigins`]..
```
Comment (adellape) [inline architecture/infrastructure_components/web_console.adoc]: Dupe period.

**Comment by adellape** [comment]
Merging, will get the minor nit in https://github.com/openshift/openshift-docs/pull/514.

**Pull request merged**